html/pfappserver/lib/pfappserver/Form/Config/Source/Eduroam.pm

```diff
@@ -126,7 +126,25 @@ has_field 'monitor',
              help => 'Do you want to monitor this source?' },
    default => pf::Authentication::Source::EduroamSource->meta->get_attribute('monitor')->default,
 );
+has_field 'testUser',
+  (
+   type => 'Text',
+   label => 'Test User',
+   element_class => ['input-small'],
+   tags => { after_element => \&help,
+             help => 'The username to use for testing when monitoring the RADIUS source' },
+   default => '',
+);
 
+has_field 'testPassword',
```

Comment:
Add testPassword to the fields_to_mask list in the sources.
https://github.com/inverse-inc/packetfence/blob/devel/lib/pf/UnifiedApi/Controller/Config/Sources.pm#L246